tests/console/check_system_info.pm

```diff
@@ -165,6 +179,15 @@ sub run {
         check_buildid;
         check_feature if (is_sle(">=15-SP4") && check_var('INSTALLONLY', '1'));
     }
+
+    # Check feature SLE-21783: Check nodejs-common on SLES15SP3 before migration
+    if (is_sle('=15-SP3') && get_var('SCC_ADDONS') =~ /wsm/) {
```

Comment:
This condition need to be updated to fulfill base system more than 15-SP3.
is_sle('>=15-SP3', get_var('ORIGIN_SYSTEM_VERSION')

Reply:
done, thanks.